satsolver.go

```diff
@@ -0,0 +1,26 @@
+package go_sat_solver
+
```

Comment:
consider leaving a comment, either at the top of the file or at the top of each method you want people to use, to tell them how to use it